plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,89 +3,98 @@ p.
+Because only transitions can produce outputs, many actions need to be duplicated. For example, from each state there is a transition @OFF_pressed / brightness = 0@ to the _Light Off_ state. These redundancies make this rather simple system already harder to read and maintain as necessary.
 
-!(large)images/ferry_harel_1.png(Ferry example as a Harel statechart)!
+Harel statecharts combine Mealy and Moore machines, hence outputs can be produced by states and transitions. Furthermore, states can be grouped together to form a hierarchy. As a Harel statechart the light switch example looks like the following:
 
-p=. Ferry example as a Harel statechart
+!(medium)images/overview_harel.png(Extended light switch example as a Harel statechart)!
 
-The *carEnter* / *carLeave* states were substituted by a composite state *FerryStops* with a substate machine that describes the behavior of the ferry at harbor. The lifts to the car deck shall be locked, because passengers must be there during transfer. There are transitions crossing the hierarchy from the substate machine to the upper-level states (inter-level transitions) like the one from state *LockLifts* to *Northwards* .
+We can even go one step further and hide the logic of the _Motion Detection Mode_ into a sub diagram. In that way, the system gets more comprehensive as one can directly see the different modes and how to switch between them.
 
-h2(#overview_uml). The present age: UML state machines
+!(medium)images/overview_harel_2.png(Extended light switch example as a Harel statechart with sub diagrams)!
 
-UML state machines are based on the statechart notation introduced by David Harel. Furthermore, the UML extends the notation of Harel statecharts by object-oriented principles. Mapping this to our ferry example, in UML we can model the possible actions of the ferry as an interface with operations stop, moveNorthwards, moveSouthwards and so on.
+Further concepts like orthogonality, entry/exit actions or history states are left out here for brevity. You can read about them in our "quick reference":../user-guide/quick_ref.html.
+
+*The present age: UML state machines*
+
+UML state machines are based on the statechart notation introduced by David Harel. Furthermore, the UML extends the notation of Harel statecharts by object-oriented principles. Mapping this to our light switch example, in UML we can model the possible actions of the light switch as a type with operations @turnOn()@, @turnOff()@, @setBrightness(value)@ and so on.
 
 The following table illustrates the differences between the previously described types at a glance:
 
 !(large)images/statemachine_types.png(Differences between state machine types)!
```

Comment:
Graphics must list concept of (extended state) variables - checked for Harel and UML

Reply:
Isn't it subsumed in the last line of the table in the term "actions"?